libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -45,7 +57,7 @@ public void onInit(int status) {
    */
```

Comment:
Could we add some Javadoc comments here and in the rest of the `public` methods (`isMuted` and `setMuted`)?